### ISSUE TITLE
Inconsistent path naming causes issues for Linux users

### ISSUE THREAD

**Issue by JohnConfidence** [2006rs/singleplayer#5]
Using Archlinux w/ Kernel 5.2.9 and Oracle JDK 8u22. Just sticks on "starting up" and doesn't go any further.

Running in the terminal yields the following error:
```java
[8/29/19 6:45 PM]: Read Error: ./Data/content/questing/Quests.dat
java.lang.NullPointerException
        at com.rs2.util.F.a(Unknown Source)
        at com.rs2.model.a.h.d.b(Unknown Source)
        at com.rs2.Server.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:748)
java.lang.ArrayIndexOutOfBoundsException: 5
        at com.rs2.model.d.b.b.a(Unknown Source)
        at com.rs2.Server.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:748)
java.lang.NullPointerException
        at com.rs2.model.d.b.b.a(Unknown Source)
        at com.rs2.Server.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:748)
```
This reveils that `./Data/content/questing/Quests.dat` would be the cause of the error; running the server for the first time generates the `data` folder. 

Linux uses case-sensitive filesystems so this would normally be solved with renaming the `data` folder to `Data`, but that results in the following error:
```java
Read Error: ./data/launcher/map_index.dat
java.lang.NullPointerException
        at com.rs2.util.F.c(Unknown Source)
        at com.rs2.model.a.j.c.o.a(Unknown Source)
        at com.rs2.launcher.p.<init>(Unknown Source)
        at com.rs2.launcher.Cpanel.<clinit>(Unknown Source)
```
As you see in this error ` ./data/launcher/map_index.dat` is unreadable because, as far as the server can tell, this file does not exist. 

This issue is 'solved' by copying the `data` directory and renaming the copy `Data`. This is not a real solution though. The best solution would be to make sure all paths have consistent naming for folders.

**Comment by Mige5** [comment]
Fixed this, available in patch after 1.6.2.

**Comment by JohnConfidence** [comment]
This issue persists in 1.6.3 
```java
[9/1/19 6:29 PM]: Starting RuneScape on /0.0.0.0:43594...
[9/1/19 6:29 PM]: Read Error: ./Data/npcs/Npc drops.dat
java.lang.NullPointerException
        at com.rs2.util.F.c(Unknown Source)
        at com.rs2.model.c.a.a.a(Unknown Source)
        at com.rs2.Server.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by Mige5** [comment]
Dang, I thought I checked all of these, I guess 1 more left to fix.

**Comment by JohnConfidence** [comment]
This is fixed in build 1.6.4
